### ISSUE TITLE
Doesn't work with OptiFine 1.15.2 HD U G1 pre16

### ISSUE THREAD

**Issue by MoustacheOff** [modmuss50/OptiFabric#196]
"OptiFabric : Can't find OptiFIne File in mods folder."

**Comment by 0xfeeddeadbeef** [comment]
Works for me.

**Comment by grind651** [comment]
Works for me too. But CTMs are not working correctly

**Comment by OakBricks-old** [comment]
Yep it crashes on 1.15.2 The game crashed "`unexpected error Error: java.lang.NullPointerException: Unexpected error`"

**Comment by OakBricks-old** [comment]
Ok, I just found out this issues is caused by shaders, to enable them, you have to disable shaders in the title menu then load and load the shaders in game.

**Comment by Username404-59** [comment]
When I start minecraft i get this: https://pastebin.com/kcYu8vrG
So yeah there's an error about shaders at the end of it.
Edit: After launching minecraft= https://imgur.com/a/dh5Pt3y

**Comment by OakBricks-old** [comment]
Try removing shaders from your shaders folder, and launch the game

**Comment by Username404-59** [comment]
It seems that for me the pre17 solved this problem.
